build/migration/bin/4_current-migrate.sh

```diff
@@ -19,7 +19,7 @@ run_migrate(){
     mvn -T2C -B install --file fhir-parent -DskipTests -P include-fhir-igs,integration --no-transfer-progress
 
     cd fhir-install
-    docker build -t ibmcom/ibm-fhir-server:snapshot .
+    docker build --build-arg VERBOSE=false -t ibmcom/ibm-fhir-server:snapshot .
```

Comment:
Do we know how long these old images will be available?

Reply:
no, we don't.  i don't think they'll go away any time soon.
that said, in this particular case, we're actually building an ibmcom/ibm-fhir-server image from source.
in all the other spots, I updated the image name.  But specifically for the migration tests, the `ibmcom/ibm-fhir-server` name is actually in our source code under fhir-install/pom.xml and so I had to chose between:
A. patching that file in the older version that we checkout so that it uses linuxforhealth/fhir-server ; or
B. continuing to using ibmcom/ibm-fhir-server for the migration tests

For now, I chose B.  We should be able to change it once we stop testing migrations from 4.x